.github/workflows/uv.yml

```diff
@@ -18,6 +18,10 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: astral-sh/setup-uv@v4
+        with:
+          enable-cache: true
+          cache-dependency-glob: "uv.lock"
       - run: uv python install ${{ matrix.python-version }}
       - run: uv sync --all-extras --dev
       - run: uv build
+      - run: uv run python -m unittest
```

Comment:
This line ouputs :
```
Ran 0 tests in 0.000s
```

Reply:
ref. https://docs.python.org/3/library/unittest.html#unittest-test-discovery:
> In order to be compatible with test discovery, all of the test files must be [modules](https://docs.python.org/3/tutorial/modules.html#tut-modules) or [packages](https://docs.python.org/3/tutorial/modules.html#tut-packages) importable from the top-level directory of the project

to allow importing `tests/test_client.py`, your `tests/` directory must be a python module. And to transform a directory into a python module, you just need to add a `__init__.py` file in there, which can be blank:
`touch tests/__init__.py`

Reply:
Ohh, ok I understand now. Thank you